core/src/main/java/com/sopt/core/designsystem/component/timepicker/NoostakTimePicker.kt

```diff
@@ -0,0 +1,333 @@
+    var selectedStartMinute by remember { mutableStateOf(0) }
+    var selectedEndHour by remember { mutableStateOf(18) }
+    var selectedEndMinute by remember { mutableStateOf(0) }
+
+    var isStartTimeEditing by remember { mutableStateOf(true) } // 시작 시간이 기본 활성화
+    var isEndTimeEditing by remember { mutableStateOf(false) }
+
+    val values = remember { (0..23).map { it.toString() } }
+    val units = remember { (0..59).map { it.toString() } }
+    val valuesPickerState = rememberPickerState()
+    val unitsPickerState = rememberPickerState()
+
+    LaunchedEffect(isStartTimeEditing, isEndTimeEditing) {
+        if (isStartTimeEditing) {
+            valuesPickerState.selectedItem = selectedStartHour.toString()
+            unitsPickerState.selectedItem = selectedStartMinute.toString()
+        } else if (isEndTimeEditing) {
+            valuesPickerState.selectedItem = selectedEndHour.toString()
+            unitsPickerState.selectedItem = selectedEndMinute.toString()
+        }
+    }
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(301.dp)
+            .background(colors.gray50, RoundedCornerShape(20.dp))
+            .padding(top = 27.dp)
+            .noRippleClickable {
+                isStartTimeEditing = false
+                isEndTimeEditing = false
+            },
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 58.5.dp),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            TimeDisplay(
+                label = stringResource(R.string.text_noostak_time_picker_start),
+                hour = selectedStartHour,
+                minute = selectedStartMinute,
+                isSelected = isStartTimeEditing,
+                onClick = {
+                    isStartTimeEditing = true
+                    isEndTimeEditing = false
+                }
+            )
+
+            Text(
+                text = stringResource(R.string.text_noostak_time_picker_wave),
+                style = typography.h1Bold,
+                color = colors.gray700
+            )
+
+            TimeDisplay(
+                label = stringResource(R.string.text_noostak_time_picker_end),
+                hour = selectedEndHour,
+                minute = selectedEndMinute,
+                isSelected = isEndTimeEditing,
+                onClick = {
+                    isStartTimeEditing = false
+                    isEndTimeEditing = true
+                }
+            )
+        }
+
+        Box(
+            modifier = Modifier
+                .padding(top = 9.dp, bottom = 30.dp, start = 33.5.dp, end = 33.5.dp)
+                .fillMaxWidth()
+                .height(3.dp)
+                .background(colors.gray100)
+        )
```

Comment:
P1: 가로선은 HorizontalDivider로 수정해주세요
```suggestion
        HorizontalDivider(
            modifier = Modifier
                .fillMaxWidth()
                .padding(top = 9.dp, bottom = 30.dp, start = 33.5.dp, end = 33.5.dp)
                .clip(CircleShape),
            thickness = 3.dp,
            color = colors.gray100
        )
```

Reply:
녜에